system/xmllib/libxml_xpathprocessor.cpp

```diff
@@ -241,15 +241,15 @@ class CLibXpathContext : public CInterface, public IXpathContext
             m_xmlDoc = xmlParseDoc((const unsigned char *)xmldoc);
             if (m_xmlDoc == nullptr)
             {
-                ERRLOG("XpathProcessor:setxmldoc Error: Unable to parse XMLLib document");
+                OERRLOG("XpathProcessor:setxmldoc Error: Unable to parse XMLLib document");
```

Comment:
Do you know the xml being processed is a system xmlfile?  No indication of what file, so almost certainly needs to be I